lib/native-package-installer/platform.rb

```diff
@@ -46,6 +46,7 @@ def detect
 
 require_relative "platform/debian"
 require_relative "platform/ubuntu"
+require_relative "platform/alpine-linux"
```

Comment:
These `require_relative`s are for Debian based distributions.
Could you put this after `platform/suse`?

```diff
diff --git a/lib/native-package-installer/platform.rb b/lib/native-package-installer/platform.rb
index 8476fbd..528c785 100644
--- a/lib/native-package-installer/platform.rb
+++ b/lib/native-package-installer/platform.rb
@@ -57,6 +57,8 @@ require_relative "platform/amazon-linux-2023"
 
 require_relative "platform/suse"
 
+require_relative "platform/alpine-linux"
+
 
 # Windows
 require_relative "platform/msys2"
```